<launch>
    <arg name="first_tb3"  default="tb3_0"/>
    <arg name="second_tb3" default="tb3_1"/>
    
    <node pkg="multi_robot_challenge" name="leader" type="leader.py" output="screen" />    

    <group ns="$(arg first_tb3)">
        <node pkg="multi_robot_challenge" name="robot" type="robot.py" output="screen" />
            <param name="robot_name" value="$(arg first_tb3)"/>
    </group>

    <!-- <group ns="$(arg second_tb3)">
        <node pkg="multi_robot_challenge" name="robot" type="robot.py" output="screen" />
            <param name="robot_name" type="string" value="$(arg second_tb3)"/>
    </group> -->
</launch>